src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -361,6 +357,10 @@ IEnumerable<Config> GetAotConfigs ()
 				if (!Directory.Exists (outdir))
 					Directory.CreateDirectory (outdir);
 
+				if (outdir.StartsWith (WorkingDirectory, StringComparison.InvariantCultureIgnoreCase)) {
```

Comment:
This bit is needed because some of the aot options still don't like spaces. However all of the files we create are in the IntermediateOuputPath. So we can strip off the `WorkingDirectory` part of the path since it won't be need (as the process is started in the `WorkingDirectory`. 

The `WorkingDirectory` is normally the root of the project where the csproj is, and that is the part which will generally contain spaces and non-ascii characters.